bot/cogs/eval.py

```diff
@@ -97,8 +97,7 @@ def _format(self, inp, out):  # (str, Any) -> (str, discord.Embed)
             res = (res, out)
 
         else:
-            if (isinstance(out, str) and
-                    out.startswith("Traceback (most recent call last):\n")):
+            if (isinstance(out, str) and out.startswith("Traceback (most recent call last):\n")):
```

Comment:
If you remove the newline, might as well drop the braces

Reply:
I tried to do the minimum possible to fix the lint errors. Maybe I should have taken the opportunity to clean up some of the code but felt like that can be done more comprehensively in a later PR.